bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/container.jsx

```diff
@@ -10,19 +10,25 @@ class MuteAudioContainer extends React.Component {
   render() {
     return (
       <MuteAudioComponent
+        isInAudio = {this.props.isInAudio}
         isMuted = {this.props.isMuted}
         muteUser = {this.props.muteUser}
         unmuteUser = {this.props.unmuteUser}
+        disabledCallback = {this.props.disabledCallback}
```

Comment:
Just a thought I think this could also be written like

``<MuteAudioComponent {...this.props} />``